baggage/HTTP_HEADER_FORMAT.md

```diff
@@ -20,7 +20,7 @@ This section uses the Augmented Backus-Naur Form (ABNF) notation of [[!RFC5234]]
 
 ```
 list        = list-member 0*179( OWS "," OWS list-member )
-list-member = key OWS "=" OWS value *( OWS ";" OWS property )
+list-member = property *( OWS ";" OWS property )
```

Comment:
this is correct from syntax perspective, but clouds the semantics a little bit. I worry implementations may start treating the whole set as a bag of properties instead of key/value pair with the additional clarifying properties. I'd go even go to differentiating `key` and `property-key` definitions to dignify importance of this differentiation.

Reply:
+1 to Sergey's comment; I prefer for this to be explicitly treated as a dictionary rather than a set of properties

Reply:
+1 to keep kv. Also, the proposed change seems simply incorrect because property can be just a key without value, but list member should not be just a key.

Reply:
I see, it makes sense to keep the current definition then.
@SergeyKanzhelev, your proposal sounds good - this would make it certainly more obvious.